### ISSUE TITLE
[Bug]: Genshin Impact does not display ReCapcha during user authentication and cannot log in

### ISSUE THREAD

**Issue by darktribe** [Whisky-App/Whisky#928]
### Description

Genshin Impact does not display ReCapcha during user authentication and cannot log in

### Steps to reproduce

Install Genshin Impact with Whisky,and Run Genshin Impact launcher,then User user authentication  run.
If you enter your username and password at that time, email authentication will be performed, but authentication cannot be performed because ReCapcha, which is required for email authentication, is not displayed and only a white screen is displayed.

### Expected behaviour

<img width="1710" alt="スクリーンショット 2024-04-14 3 16 28" src="https://github.com/Whisky-App/Whisky/assets/38865226/5a896ac2-1757-4b56-bc67-76ab6c134b95">

I want to display ReCapcha, which is necessary for email authentication.

### Logs

```bash
Whisky Version: 2.3.2
Date: 2024-04-13T18:15:48Z
macOS Version: 14.4.1

Bottle Name: Windows10
Bottle URL: /Users/show-ya/Library/Containers/com.isaacmarovitz.Whisky/Bottles/B81904B6-6BAC-478D-A4D7-3A8FA21CCAEF

Wine Version: 7.7.0
Windows Version: win11
Enhanced Sync: msync

Metal HUD: true
Metal Trace: true

DXVK: true
DXVK Async: true
DXVK HUD: off

Arguments: start /unix /Users/show-ya/Library/Containers/com.isaacmarovitz.Whisky/Bottles/B81904B6-6BAC-478D-A4D7-3A8FA21CCAEF/drive_c/Program Files/Genshin Impact/Genshin Impact game/GenshinImpact.exe

Environment:
{
    "DXVK_ASYNC" = 1;
    "GST_DEBUG" = 1;
    "LC_ALL" = "ja_JP.UTF-8";
    "METAL_CAPTURE_ENABLED" = 1;
    "MTL_HUD_ENABLED" = 1;
    WINEDEBUG = "fixme-all";
    WINEDLLOVERRIDES = "dxgi,d3d9,d3d10core,d3d11=n,b";
    WINEESYNC = 1;
    WINEMSYNC = 1;
    WINEPREFIX = "/Users/show-ya/Library/Containers/com.isaacmarovitz.Whisky/Bottles/B81904B6-6BAC-478D-A4D7-3A8FA21CCAEF";
}
```

### What version of Whisky are you using?

2.3.1

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by smithc** [comment]
Looks like this may have been solved by another user in a previously opened issue: https://github.com/Whisky-App/Whisky/issues/858#issuecomment-1987155593

> For blank puzzle, try source-han-sans fonts.
> 
> Download and Install fonts for your macOS. [Language Specific OTFs Japanese (日本語)](https://github.com/adobe-fonts/source-han-sans/releases/download/2.004R/SourceHanSansJ.zip) ([source](https://github.com/adobe-fonts/source-han-sans/releases/tag/2.004R))
> 
> Launch Whisky and open Registry Editor, press control + F to search "fontsub", modify content of both "MS Shell Dlg" and "MS Shell Dlg 2" to "源ノ角ゴシック".
> 
> Idea from a [YouTube](https://youtu.be/0hgZFFKS_1A) video with same problem in Traditional Chinese fonts.

**Comment by Nuanulla** [comment]
I was able to make the captcha pop up without any winetricks or other wine/whisky workarounds (as in, no regedits either). I will note that I have Pinyin Traditional and Japanese keyboard input sources enabled under System Settings (in addition to ABC), but that shouldn't make a difference. I've consistently run Genshin Impact on macOS 14.4.1 and Whisky 2.3.2. Default bottle settings.

**Comment by baiyu128** [comment]
> I was able to make the captcha pop up without any winetricks or other wine/whisky workarounds (as in, no regedits either). I will note that I have Pinyin Traditional and Japanese keyboard input sources enabled under System Settings (in addition to ABC), but that shouldn't make a difference. I've consistently run Genshin Impact on macOS 14.4.1 and Whisky 2.3.2. Default bottle settings.
> 
> Edit: I should note that the one issue I've consistently found is that, if Genshin is run in Fullscreen and the user tabs out or switches Desktop spaces, the Genshin window will no longer accept user keyboard inputs once the wine window is back in focus. The game requires Borderless Fullscreen or Windowed mode to avoid this issue. I mention this as it may or may not be related to the blank captcha.

Can I run 4.6 version normally now? My process cannot start entering the game after loading. I am stuck at that door. @Nuanulla

**Comment by Nuanulla** [comment]
@baiyu128 yeah you should be able to. Unless you need to download the fonts that smithc linked due to region differences in the macs we purchased.
